### PR TITLE
MCNP precision

### LINKED ISSUE #1203: increase precision for "C density" comment in write_mcnp
The method write_mcnp creates an MCNP material card from a PyNE material with a comment line indicating the density.  Currently it only provides a precision of 1.  For example:
C density = 6.3
m1
     2003 -5.6948e-12
     2004 -5.6400e-06
...
It would be helpful for quality assurance and reviewing purposes to increase the precision to at least 3 and perhaps as high as 5 or 6 depending on the application of interest for the MCNP calculation.

### PR DESCRIPTION
**Pull request by gonuke**
Fixes #1203 

Changes the precision reported in the MCNP comment to have 5 decimal places.

### CODE REVIEW

**Comment by bam241** [comment]
I was doing it, got caught up in meetings... tests need to be fixed as well...
CI did not run there, weird...

File: news/mcnp_precision.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:** None
```
Comment (bam241) [inline news/mcnp_precision.rst]:
```suggestion
**Changed:**
```

**Review by bam241 — CHANGES_REQUESTED** [review]
Just a minor correction, 
thx @gonuke

**Review by zxkjack123 — APPROVED** [review]
LGTM.

**Review by bam241 — APPROVED** [review]
thx @gonuke 

merging now !

**Pull request merged**